### ISSUE TITLE
Issue with pyside

### ISSUE THREAD

**Issue by elmerehbi** [lordamit/Brightness#67]
Running `src/init.py` works while adding python doesn't & returns this error through pyside was installed:

```
python src/init.py
Traceback (most recent call last):
  File "src/init.py", line 19, in <module>
    from PySide import QtGui, QtCore
ImportError: No module named PySide

```

Might an Anaconda Python be the cause?

Confirmed. Installing pyside with apt-get doessn't work if you have Anaconda. Instead using `conda install pyside` or `pip install pyside` should work.

**Comment by lordamit** [comment]
I am unfamiliar with the term Anaconda. A quick google search shows that it is probably a python distribution focusing on analytics.

I will update the readme and add instructions of installing PySide through pip :smile: 

Thank you for pointing this out.

**Comment by apandada1** [comment]
I could not understand the problem. Please explain

**Comment by lordamit** [comment]
Just that sudo apt-get install python-pyside was not enough for his system to use the app.

**Comment by elmerehbi** [comment]
@apandada1 Anaconda as @lordamit  mentioned is a Python distribution packaged for scientists and analysts. It installs in the users home directory and changes Python's path to Anacondas' python (in .bashrc, or zshrc, etc...) SO basically when I am calling python I am using the Anaconda Python and its packages.

Using `apt-get` will install the packages in the original Python directories which are not being used anymore. Instead `pip` and `conda` (latter is Anaconda's package manager) install the package in the Anaconda python library directories.

IN short: the `apt-get` install doesn't get used.

**Comment by lordamit** [comment]
@elmerehbi , I have updated the Readme.md file. If you the instructions are better now and solves this problem - close this issue maybe?

**Comment by apandada1** [comment]
Is there any command to force regular Python?
It is not easy to add the following feature to deb package:
If Anaconda Python is installed : pip install pyside after installation, (this will require adding a bash script)
otherwise, apt-get install pyside

**Comment by elmerehbi** [comment]
@apandada1 @lordamit I get you. You could replace `apt-get install pyside` with `pip install pyside` as in the case of Anaconda the path to Python is changed and hence the new pip is going to be used. No need to use `conda`.

So instead of the system's package manager (`apt-get` here) use Python's package manager `pip`

**Comment by apandada1** [comment]
No, that's not the solution.
I will have to add dependency in the "control" file in the .deb package. 
![screenshot from 2016-04-29 13-08-00](https://cloud.githubusercontent.com/assets/3063132/14910571/346dac78-0e0c-11e6-9296-9ad63d351d5a.png)


The dependencies asked here will be installed via apt (dpkg). We cannot use pip here.

What I wanted to know: Is there any specific command which will invoke original Python?
Suppose `python --option src/init.py`?

Or, can it be done by replacing `#!/usr/bin/python` in the init.py file?

**Comment by elmerehbi** [comment]
I just checked the init.py & it includes /usr/bin/python. So I think there's no need to do anything as it points to the system's Python distribution. The only thing that caused a problem is calling `python` before the script:

```
python /src/init.py # returns the error I posted above
./src/init.py # works like a charm
```
Moreover, since I've got the ppa added I have the older version working too without the need for pyside in Anaconda's python.

**Comment by lordamit** [comment]
@apandada1 , we are talking about the version 2 here. 
What we can do, is create a checking mechanism for the existence of Pyside in user's python distribution when the app is run for the first time after installation through the PPA, and then automatically install python-pyside using pip if required. Or ask user to install Python-pyside using pip.

**Comment by elmerehbi** [comment]
@apandada1 Doesn't running it with `./` suffice instead of explicitly calling python given that the python path is already set in the script?

**Comment by apandada1** [comment]
We can mark it as executable and directly run it (without `python init.py`) (that's how v1 PPA works too) . Then there will be no need of checking mechanism, and the problem will be fixed

**Comment by lordamit** [comment]
The **App GUI** runs using PySide, but the app itself can be loaded from a separate script which is not dependent on PySide. That part will execute just like any other Python script.

**Comment by elmerehbi** [comment]
If that' ok I suggest it instead of worrying about which python is installed given Python is alredy installed and its path is known.

Otherwise I don't see why it would be a problem to install with pip. Youd on't even need `apt-get` which is Debian specific while `pip` is Python & **user** specific. So instead of `sudo apt-get install ...` just replace it with `pip install ...`. This will take care of the dependency & install it according to the user's Python path.
If you do this I think you will have to remove the shebang line in your scripts and then you can call `python <your scripy>`.

**Comment by apandada1** [comment]
@elmerehbi When user installs it manually, he can run pip install pyside. But a .deb package (which will be in the PPA) cannot run pip commands, it resolved dependencies using apt-get. The only simple solution is to run the app with usr/bin/python

**Comment by elmerehbi** [comment]
Ah ok. Indeed /usr/bin/python should work.
